services/device-registry-mongodb/src/test/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedCredentialServiceTest.java

```diff
@@ -170,44 +169,12 @@ public CredentialsService getCredentialsService() {
 
     @Override
     public CredentialsManagementService getCredentialsManagementService() {
-        return this.credentialsService;
+        return this.credentialsManagementService;
     }
 
     @Override
     public DeviceManagementService getDeviceManagementService() {
-        return this.deviceBackendService;
-    }
-
-    /**
-     * Verifies that a request to add credentials for a device fails with a 403 status code
-     * if the number of credentials exceeds the tenant's configured limit.
-     *
-     * @param ctx The vert.x test context.
-     */
-    @Test
-    public void testAddCredentialsFailsForExceededCredentialsPerDeviceLimit(final VertxTestContext ctx) {
```

Comment:
Don't we need this test anymore?

Reply:
I don't think so. The MongoDbBasedCredentialsManagementService does not have an *addCredentials* method anymore and when we create a device, an empty set of credentials is being created ...